### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amyskov**
Minor corrections of README.md file Windows sections were made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
 ### Required for Windows:
 * `python` installed
 * `cmake`
-* Visual Studio compilers, version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
+* Visual Studio with release year not older than 2015 and Visual Studio compiler, which version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
```
Comment (vnlitvinov) [inline README.md]: why change? I don't see added value by this change...

File: README.md [hunk 2]
```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
...
 
 
 ### Before building do:
 * On Linux:
   * `git submodule update --init --recursive`
   * `cd 3rd_party && ./prereq-build.sh && cd ..`
 * On Windows:
-  * `git submodule update --init --recursive`
-  * `cd 3rd_party/protobuf-c && git apply ../protobuf-c-vs2008-support.patch && cd ../..`
+  * Update third-party intel-xed, mbuild, protobuf, protobuf-c and safestringlib submodules (3rd_party folder):
```
Comment (vnlitvinov) [inline README.md]:
```suggestion
  * Update third-party intel-xed, mbuild, protobuf, protobuf-c and safestringlib submodules (`3rd_party` folder):
```

File: README.md [hunk 3]
```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
...
+      * `git submodule update --init -- .\3rd_party\intel-xed`
+      * `git submodule update --init -- .\3rd_party\mbuild`
+      * `git submodule update --init -- .\3rd_party\protobuf`
+      * `git submodule update --init -- .\3rd_party\protobuf-c`
+      * `git submodule update --init -- .\3rd_party\safestringlib`
+  * Apply protobuf-c-vs2008-support.patch:
```
Comment (vnlitvinov) [inline README.md]: add a note that this is required for Python 2 target only (not needed for Python 3)

File: README.md [hunk 4]
```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
...
+      * `cd 3rd_party\protobuf-c && git apply ..\protobuf-c-vs2008-support.patch && cd ..\..`
+  * If you have installed VS2017 or VS2019, apply next command (optional step), please note, that path to the vcvars64.bat file may vary by version of VS installed:
```
Comment (vnlitvinov) [inline README.md]: where do you want a user to "apply" that? previous commands could be executed in any order in any terminal as they're persistent, this one modifies the environment variables of single command line prompt window

File: README.md [hunk 5]
```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
...
+      * `CALL "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build\vcvars64.bat" -vcvars_ver=14.0`
   * For Python 2.x run:
-    * `cd 3rd_party && prereq-build-py2.cmd && cd ..`
+      * `cd 3rd_party && prereq-build-py2.cmd && cd ..`
```
Comment (vnlitvinov) [inline README.md]: why change indentation?

File: README.md (hunk 1 above)
Comment (amyskov) [reply README.md]: This change is confusing and not necessary, change was canceled in new commit.

File: README.md (hunk 3 above)
Comment (amyskov) [reply README.md]: Ok, comment was addressed in new commit.

File: README.md (hunk 4 above)
Comment (amyskov) [reply README.md]: You are right, step description is confusing, comment is addressed in new commit.

File: README.md (hunk 5 above)
Comment (amyskov) [reply README.md]: Indentation change is not necessary, change was canceled in new commit.

**Comment by amyskov** [comment]
@vnlitvin , your comments were addressed, please review.

File: README.md (hunk 2 above)
Comment (vnlitvinov) [reply README.md]: It seems you didn't apply my suggestion here, why?

**Review by vnlitvinov — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (amyskov) [reply README.md]: Suggestion was rejected by mistake, now quotes are added.

**Pull request merged**